src/WireMock.Net.OpenApiParser/Settings/WireMockOpenApiParserSettings.cs

```diff
@@ -36,5 +36,33 @@ public class WireMockOpenApiParserSettings
         /// Are examples generated dynamically?
         /// </summary>
         public bool DynamicExamples { get; set; } = false;
+
+        /// <summary>
+        /// Is headers case sensitive? (default is true).        
+        /// true = no case sensitive;
```

Comment:
No need to add these two lines:
```
        /// true = no case sensitive;
        /// false = case sensitive 
```

You can remove these.

Reply:
@StefH  Done!